terraform/global/um-pep-engine/config/pep-engine.tf

```diff
@@ -3,8 +3,8 @@ resource "kubernetes_config_map" "pep_engine_cm" {
     name = "um-pep-engine-config"
   }
 
-  depends_on = [null_resource.waitfor-login-service]
 
+  depends_on = [null_resource.waitfor-login-service]
```

Comment:
This line might no longer be necessary thanks to:
module_depends_on = [module.um-login-service] on the main.tf file